control-plane/mbus-api/src/v0.rs

```diff
@@ -1161,22 +1246,26 @@ pub struct CreateVolume {
     pub uuid: VolumeId,
     /// size of the volume in bytes
     pub size: u64,
-    /// number of children nexuses (ANA)
-    pub nexuses: u64,
-    /// number of replicas per nexus
+    /// number of storage replicas
     pub replicas: u64,
-    /// only these nodes can be used for the replicas
-    #[serde(default)]
-    pub allowed_nodes: Vec<NodeId>,
-    /// preferred nodes for the replicas
-    #[serde(default)]
-    pub preferred_nodes: Vec<NodeId>,
-    /// preferred nodes for the nexuses
-    #[serde(default)]
-    pub preferred_nexus_nodes: Vec<NodeId>,
+    /// replica replacement policy
+    pub policy: VolumeHealPolicy,
```

Comment:
The comment mentions replica but the type is volume which is a little confusing. I think the comment should say volume. The user can set the policy on the volume, but internally we manage it by manipulating the replicas.

Reply:
Yep I had it all as replica then replaced with volume but clearly missed some :)